olm/open-liberty-crd.yaml

```diff
@@ -373,8 +375,10 @@ spec:
                   properties:
                     cpu:
                       type: string
+                      pattern: '^[0-9]+m?$'
                     memory:
                       type: string
+                      pattern: '^[0-9]+([kKmMgGtTpPeE][i]?|[e][0-9]*)?$'
```

Comment:
This looks fine, but just one observation that the doc (https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) states that these suffixes be used: E, P, T, G, M, K. Your expression allows for lower case values such as 100gi, or 10e (10 exa bytes).

Reply:
Will update to remove the mixed-case support.